src/pages-and-resources/discussions/Discussions.jsx

```diff
@@ -0,0 +1,150 @@
+import React, {
+  useCallback, useEffect, useRef, useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import {
+  Switch,
+  useLocation,
+  useRouteMatch,
+} from 'react-router';
+import { history } from '@edx/frontend-platform';
+import { PageRoute } from '@edx/frontend-platform/react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Button, StatefulButton } from '@edx/paragon';
+import { Check } from '@edx/paragon/icons';
+
+import FullScreenModal from '../../generic/full-screen-modal/FullScreenModal';
+import Stepper from '../../generic/stepper/Stepper';
```

Comment:
Nit: Should this block of imports be grouped below with the other internal project imports?

Reply:
I sometimes end up putting a line break between different groups in a somewhat subjective way... these were generic and ultimately belong in Paragon, so they got their own grouping. :) I don't feel super strongly at all.